pkgs/top-level/release-small.nix

```diff
@@ -33,20 +33,17 @@ with import ./release-lib.nix { inherit supportedSystems nixpkgsArgs; };
   classpath = linux;
   cmake = all;
   coreutils = all;
-  cpio = all;
```

Comment:
isn't that useful to deal/debug initramfs ? could be useful

Reply:
It seemed to have a lot of overlap with `gnutar`. And `libarchive` seems to one-up it in almost every use case

Reply:
ok trust you with this :)

Reply:
These assumptions are based upon https://www.gnu.org/software/cpio/

I haven't actually heard of the tool before today.